docs/content/agents/agent-product-search.mdx

```diff
@@ -2,10 +2,34 @@ import AgentPlugin from "@/components/agent-plugin";
 
 export const description = "Product Search Agent helps users find the right products effortlessly, even without prior knowledge of the domain. By understanding user needs through natural language queries, it recommends relevant products tailored to individual preferences. The agent intelligently filters options, highlights key features, and simplifies comparisons, saving users time and effort. It’s designed to make product discovery intuitive, accurate, and personalized, turning complex searches into simple conversation.";
 
+export const useCases = [
+  {
+    title: "Marketplace search enhancement",
+    description: "Improves search experience across marketplaces by understanding natural language queries. Provides accurate product matches even with vague or partial inputs."
+  },
+  {
+    title: "Sales support for non-experts",
+    description: "Allows sales representatives without deep product knowledge to find relevant products quickly. Reduces training time and speeds up customer response."
```

Comment:
The description is concise but could be more impactful by focusing on how it achieves this and the benefits in measurable terms.

```suggestion
    description: "Enables sales representatives without deep product knowledge to quickly identify relevant products, reduces training requirements, and reduces customer response time."
```